### PR TITLE
Updated license and readme

### PR DESCRIPTION
**Pull request by blackshadowshade**
I noticed that our LICENSE and README files were out of date, so I decided to update them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ buttonmen
 
 Buttonmen - an online dice game
 
-Development forum: http://z13.invisionfree.com/ButtonMenOnlineDev/
+Come and play with us at http://www.buttonweavers.com/
```
Comment (cgolubi1) [inline README.md:6]: Suggest rewording a little, maybe to "Play Buttonmen at http://www.buttonweavers.com/"

I don't feel strongly; overall this change seems fine.

**Comment by blackshadowshade** [comment]
Wording changed, rebased.
